yateto/codegen/visitor.py

```diff
@@ -225,7 +225,7 @@ def kernelArgs(base_name_with_namespace, groups, writable):
             header.functionDeclaration(executeName(index))
 
         if familyStride is not None:
-          header('typedef void ({}::* const {})(void);'.format(name, self.MEMBER_FUNCTION_PTR_NAME))
+          header('using {} = void ({}::*)();'.format(self.MEMBER_FUNCTION_PTR_NAME, name))
```

Comment:
Bringt das irgendetwas oder ist das wieder Geschmackssache? Nur aus Neugier, mir ist es wurscht.

Reply:
Both: The old version gives a warning with gcc8+ (ignored type modifier on return type or something similar, from the const). And the new way is a bit easier to read, at least for me:
`using member_function_ptr = void (projectToNodalBoundaryRotated::*)();` vs
 `typedef void projectToNodalBoundaryRotated::* const member_function_ptr(void)`